server/game/gamesteps/triggeredabilitywindow.js

```diff
@@ -26,7 +26,7 @@ class TriggeredAbilityWindow extends BaseAbilityWindow {
     continue() {
         this.players = this.filterChoicelessPlayers(this.players || this.game.getPlayersInFirstPlayerOrder());
 
-        if(this.players.length === 0 || this.abilityChoices.length === 0) {
+        if(this.players.length === 0) {
```

Comment:
Removing the abilityChoices check here may lead to empty prompts that you have to explicitly pass other than timed ones. Not 100% sure.